mkdocs.yml

```diff
@@ -106,6 +106,7 @@ nav:
   - 'Services':
     - services/index.md
     - 'FirecREST': services/firecrest.md
+    - 'Developer Portal': services/devportal.md
```

Comment:
Perhaps "Developer Portal" should come first?

In any case, could you add an entry to the services index page for "Developer Portal" as well?

Reply:
Done.
I have a question: I think FirecREST should be listed as an `Access method` and not in `Services`. Who manages this kind of categorization?

Reply:
Do you think it'd be worth listing it in both services and connecting to cscs (with the main home being the latter)?